.github/workflows/nightly_nosmp.yml

```diff
@@ -0,0 +1,119 @@
+name: Tests with SMP support (nightly)
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - staging
+      - trying
+
+jobs:
+  build:
+    name: Tests without SMP support
+    runs-on: ${{ matrix.os }}
+    # these tests based on the nightly compiler, which can be broken
+    # consequently, we continue on an error
+    #continue-on-error: true
```

Comment:
Should this be enabled? If not, can you remove it?

Reply:
I removed these lines.